src/test/java/software/aws/neptune/opencypher/OpenCypherDatabaseMetadataTest.java

```diff
@@ -169,4 +169,15 @@ void testGetTableTypes() throws SQLException {
         Assertions.assertEquals("TABLE", resultSet.getString(1));
         Assertions.assertFalse(resultSet.next());
     }
+
+    @Test
+    void testGetTypeInfo() throws SQLException {
```

Comment:
These tests don't really cover much... However I understand that making a test for all the different types would be time consuming.

Reply:
I think testing that getting a container type works and has something in it is sufficient.

Testing what is inside a container is relatively meaningless in my opinion since you're effectively statically writing the same thing twice.